### PR TITLE
N30: Rusty Engine 4.0

### PR DESCRIPTION
**Pull request by CleanCut**
Part of https://github.com/rust-gamedev/rust-gamedev.github.io/issues/912

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/030/index.md [hunk 1]
```diff
@@ -70,6 +70,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+![An example Rusty Engine game](rusty_engine4.png)
+_The updated collider visualization (white outlines) in Rusty Engine 4.0_
+
+[Rusty Engine] by [Nathan Stocks] is a game engine built on top of Bevy
+for people who are learning Rust.
+
+Notable new features in Version 4.0 include: no need for an `init!` macro, new
+collider visualization, text can now be rotated and scaled, and an updated
+[online tutorial]. See [the changelog for 4.0] for the full details. On the
+back end Bevy has been updated to 0.6 and we switched from
```
Comment (ozkriff) [inline content/news/030/index.md:84]: > we

could you switch to a third-person perspective, please?

File: content/news/030/index.md [hunk 2]
```diff
@@ -70,6 +70,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Engine Updates
 
+### [Rusty Engine 4.0]
+
+![An example Rusty Engine game](rusty_engine4.png)
```
Comment (ozkriff) [inline content/news/030/index.md:75]: > +623 KB 

The image is too big - could you please reduce its size at least to 300kb?

**Review by ozkriff — CHANGES_REQUESTED** [review]
LGTM in general but with a few nitpicks

File: content/news/030/index.md (hunk 2 above)
Comment (CleanCut) [reply content/news/030/index.md:75]: Shoot, I forgot to check. Yes, I'll fix it right up. 👍🏻

Comment (CleanCut) [reply content/news/030/index.md:75]: ✅ Done!

File: content/news/030/index.md (hunk 1 above)
Comment (CleanCut) [reply content/news/030/index.md]: ✅ Done!

**Review by ozkriff — APPROVED** [review]

**Comment by CleanCut** [comment]
✅ I resolved the merge conflict

**Pull request merged**